titan/agent.py

```diff
@@ -109,6 +109,14 @@ def __ne__(self, other) -> bool:
     def __hash__(self) -> int:
         return self.id
 
+    @property
+    def age_bin(self):
+        for key, val in self.location.params.classes.age_bins.items():
+            if self.age >= val.min_age and self.age <= val.max_age:
+                return key
+        # If not in an established bin, return "next" bin
+        return key + 1
```

Comment:
I don't think this will work in the case of an agent below the minimum age bin.  Instead of a bin having a min and max, would it make more sense for it to only have a max?